diagnostic_aggregator/src/aggregator_node.cpp

```diff
@@ -34,34 +34,21 @@
 
 /**< \author Kevin Watts */
 
-#include <exception>
-
 #include "diagnostic_aggregator/aggregator.hpp"
 
-#include "rclcpp/executors.hpp"
-
-using std::exception;
+#include <memory>
+#include <rclcpp/rclcpp.hpp>
```

Comment:
```suggestion

#include "rclcpp/rclcpp.hpp"
```

Reply:
Would you happen to know if there is a .clang-format file that follows your pipeline checks?

Reply:
I think you find it in the `ament_clang_format` package resp. in your ROS 2 workspace in `src/ament/ament_lint/ament_clang_format/ament_clang_format/configuration/.clang-format`